lib/iris/tests/integration/test_PartialDateTime.py

```diff
@@ -34,7 +34,6 @@ def test_netcdftime_interface(self):
         # The `netcdf4` Python module introduced new calendar classes by v1.2.7
         # This test is primarily of this interface, so the
         # final test assertion is simple.
-        with iris.FUTURE.context(cell_datetime_objects=True):
```

Comment:
Again, no unindent. Have you confirmed this test still passes? Have you considered what this test now needs to tell us?